packages/react/src/components/AccountSettings/ChangePassword/ChangePassword.tsx

```diff
@@ -204,13 +205,19 @@ function ChangePassword({
           onChange={handleChange}
           validationErrors={validationError?.confirmPassword}
         />
-        <DefaultSubmitButton isDisabled={isDisabled} type="submit">
+        <SubmitButton isDisabled={isDisabled} type="submit">
           {updatePasswordText}
-        </DefaultSubmitButton>
-        {errorMessage ? <DefaultError>{errorMessage}</DefaultError> : null}
+        </SubmitButton>
+        {errorMessage ? <Error>{errorMessage}</Error> : null}
```

Comment:
nit: Maybe worth making a `DefaultErroMessage` component that takes care of the conditional rendering?